SocialFilter/sections/specific.txt

```diff
@@ -705,7 +700,6 @@ kp.ru##.sharing_wrapper
 pdf.movavi.ru##.article-buttons
 blacksilk.sclub.com.tw##div[onmouseover^="SHARE_"]
 blog.remarkable.com##button > svg[width="29"][height="29"]
-podrobnosti.ua##.twitter-timeline
```

Comment:
`podrobnosti.ua##.twitter-timeline` is required
![image](https://user-images.githubusercontent.com/8361299/112495990-1066a680-8d8d-11eb-856f-ed54e7f368e1.png)

Reply:
The same rule is already in the common section.

Reply:
social locker rules better leave in that group

Reply:
The rule is not in the social locker group?